opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -66,6 +66,8 @@
 
 from opentelemetry import loader
 
+ParentSpan = typing.Optional[typing.Union['Span', 'SpanContext']]
```

Comment:
These type aliases are helpful but may be confusing if you'd expect this to be a real type, we may want to quarantine these, e.g. in `types.py`.

Reply:
My vote would be to leave it in the module. someone could mistakenly import it from this module anyway, and having the type definition live close to the relevant code is valuable from a refactoring and context perspective.

Reply:
Wondering if we should have this in the API, even if in a helper module (could be of help to somebody implementing the API, besides this SDK). Something to consider in the future.